data/dataloader.py

```diff
@@ -4,27 +4,30 @@
 
 """
 # Standard dist imports
-
-# Third party imports
+from collections import OrderedDict
 import os
+import sys
 
+# Third party imports
 import numpy as np
 import pandas as pd
+import PIL
+from PIL import Image
 import torch
 import torch.utils.data as data
 import torchvision.transforms as transforms
-from PIL import Image
 from torch.autograd import Variable
 from torch.utils.data import Dataset
 
-from data.prepare_db import create_proro_csv, prepare_db, create_lab_csv
 # Project level imports
+from data.prepare_db import create_proro_csv, prepare_db, create_lab_csv
 from utils.config import opt
 from utils.constants import *
 
 # Module level constants
 #TODO make this dictionary dynamic according to the data loaded
-CLASSES = {0: 'Non-Prorocentrum', 1: 'Prorocentrum'}
```

Comment:
Can you finish this todo ? Let's make the classes dynamic. Its a small thing. So, we will forget to do it later. Let's do it now

Reply:
Sorry, forgot to delete the TODO. For training, CLASSES will load automatically from train.csv, and save the CLASSES as txt file into the model_dir. Thus, when you are deploying or validating, CLASSES will be retrieved from the model_dir.